packages/mdl-button/mdl-button.scss

```diff
@@ -124,10 +128,12 @@
   &--accent {
     color: var(--mdl-theme-color-accent);
 
+    /* postcss-bem-linter: ignore */
     &.mdl-button--raised {
       background-color: var(--mdl-theme-color-accent);
       color: rgba(0, 0, 0, .87);
 
+      /* postcss-bem-linter: ignore */
```

Comment:
It's not. BEM doesn't want you combining classes. It wants a single-class target to everything.